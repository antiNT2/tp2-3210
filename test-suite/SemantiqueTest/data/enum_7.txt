bool var;
enum A {
    B, C, D
}
A var;