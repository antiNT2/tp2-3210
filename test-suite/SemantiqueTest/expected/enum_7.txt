Identifier var has multiple declarations